store/sql_channel_store.go

```diff
@@ -25,7 +25,8 @@ func NewSqlChannelStore(sqlStore *SqlStore) ChannelStore {
 		table.ColMap("DisplayName").SetMaxSize(64)
 		table.ColMap("Name").SetMaxSize(64)
 		table.SetUniqueTogether("Name", "TeamId")
-		table.ColMap("Description").SetMaxSize(1024)
+		table.ColMap("Header").SetMaxSize(1024)
+		table.ColMap("Purpose").SetMaxSize(128)
```

Comment:
The ticket only specified 100 characters although I bumped it up to 128 to keep the trend of powers of 2 for the field sizes. I assume that's because purpose is just shown in the More Channels menu so we can only fit a certain amount.